docs/shared/reference/nx-json.md

```diff
@@ -86,6 +86,24 @@ To learn about creating your own plugin read about [extending Nx](/extending-nx/
 }
 ```
 
+### Restrict Plugin Inferred Tasks
+
+You can limit which projects are processed by Nx plugins to infer tasks using the `include` and `exclude` properties in the plugin configuration object.
```

Comment:
```suggestion
You can scope which files are processed by Nx plugins to infer tasks using the `include` and `exclude` properties in the plugin configuration object.
```

I also think the section above can go into a little more details about how plugins work in general.

Specifically that they analyze files. Without that preface, i feel this section is kind of out of the blue talking about files getting processed or not processed and it's confusing.